source/includes/_aviate-catalog-apis.md

```diff
@@ -69,6 +70,43 @@ Represents a price. It has the following attributes:
 
+### UsageData
+
+Represents a usage. It has the following attributes:
+
+| Name                | Type             | Generated by | Description                                                                                                                                                                     |
+|---------------------|------------------|--------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **usageName**       | string           | user         | Usage name                                                                                                                                                                      |
+| **usageType**       | string           | user         | Usage type (One of `CONSUMABLE/CAPACITY`. Only `CONSUMABLE` supported at the time of writing)                                                         |
+| **prettyName**      | string           | user         | Pretty name for the usage                                                                                                                                                       |
+| **billingPeriod**   | string           | user         | Recurring billing period. One of the value listed [here](https://github.com/killbill/killbill-api/blob/master/src/main/java/org/killbill/billing/catalog/api/BillingPeriod.java) |
+| **tierBlockPolicy** | string           | user         | The tier block policy (One of `ALL_TIER/TOP_TIER`). Applicable only for `CONSUMABLE` usage type.                                                                                |
+| **tiers**           | List of TierData | user         | Usage price tiers                                                                                                                                                               |
+
+### TierData
+
+Represents a usage tier. It has the following attributes:
+
+| Name           | Type              | Generated by | Description |
+|----------------|-------------------|--------------|-------------|
+| **tierNumber** | int               | user         | Tier number |
+| **blocks**      | List of BlockData | user         | Tier blocks |
+
+### BlockData
+
+Represents a tier block. It has the following fields:
+
+| Name                 | Type              | Generated by | Description         |
+|----------------------|-------------------|--------------|---------------------|
+| **billingMeterCode** | string            | user         | Billing meter code  |
+| **size**             | int               | user         | Block size          |
+| **max**              | int               | user         | Max value in block  |
+| **prices**           | List of PriceData | user         | Usage price details |
+
+<!---
+Add a link to billingMeterCode resource after it is available
```

Comment:
It is indeed important to have a reference here: The endpoint already exist and has been [documented](https://github.com/killbill/killbill-aviate-plugin/blob/main/openapi/metering_paths.yml#L32), should we simply add it as part of this PR to make it consistent?

Reply:
I think I got the answer with this PR - I had not seen it. https://github.com/killbill/slate/pull/324